controllers/secretproviderclasspodstatus_controller_test.go

```diff
@@ -21,9 +21,9 @@ import (
 	"sync"
 	"testing"
 
+	"github.com/onsi/gomega"
```

Comment:
Revert this change (imports should remain grouped the way they are currently grouped)